src/components/Collection/Collection.js

```diff
@@ -133,6 +133,31 @@ const Collection = () => {
     return <PhotoBox key={item.id} item={item} />;
   }
 
+  const genericSort = [
+    {
+      sortBy: "asc",
+      dataField: "title.primary.keyword",
+      label: "Sort By Title"
+    }
+  ];
+  const searchSort = [
+    {
+      sortBy: "asc",
+      dataField: "_score",
+      label: "Sort By Relevancy"
+    },
+    {
+      sortBy: "asc",
+      dataField: "modified_date",
+      label: "Sort By Modified Date"
+    },
+    {
+      sortBy: "asc",
+      dataField: "title.primary.keyword",
+      label: "Sort By Title "
```

Comment:
I added a space here for label after "Title " due to a bug. It will sort the order correctly but it is not changing the select value on sort field if the label is same on both sort options.

<img width="944" alt="Screen Shot 2020-03-30 at 3 02 53 PM" src="https://user-images.githubusercontent.com/14085957/77969438-576fa800-72af-11ea-9f0c-8fd29044337d.png">